application/templates/base.html

```diff
@@ -1,15 +1,12 @@
 <!DOCTYPE html>
-<!--[if lt IE 9]><html class="lte-ie8" lang="{{ html_lang|default('en') }}"><![endif]-->
-<!--[if gt IE 8]><!--><html lang="{{ html_lang|default('en') }}"><!--<![endif]-->
+<html lang="{{ html_lang|default('en') }}">
   <head>
     <meta charset="utf-8">
     <title>{% block page_title %}GOV.UK - The best place to find government services and information{% endblock %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
     <!--[if IE 6]><link href="/static/stylesheets/govuk-template-ie6.css?0.19.2" media="screen" rel="stylesheet" /><![endif]-->
     <!--[if IE 7]><link href="/static/stylesheets/govuk-template-ie7.css?0.19.2" media="screen" rel="stylesheet" /><![endif]-->
```

Comment:
Also worth removing ie6/ie7 support as well?

Reply:
@samuelhwilliams yeah? I just wanted to double-check that we're not required to support these at all first.

Reply:
Is there a specific thing that says we don't need to support IE8 but doesn't mention earlier versions?  Feels like if we're not support 8 we should definitely be able to ditch 6 and 7.

Reply:
@TheDoubleK oh, my bad, I misread these as later versions (9 and 10), not earlier. We definitely don't need to support IE 6 and 7! According to [the guidance](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices) we should be testing in IE11.